tests/openapi/run.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+set -eu
+
+cur=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+source $cur/../_utils/test_prepare
+
+WORK_DIR=$TEST_DIR/$TEST_NAME
+
+function prepare_database() {
+	run_sql 'DROP DATABASE if exists openapi;' $MYSQL_PORT1 $MYSQL_PASSWORD1
+	run_sql 'CREATE DATABASE openapi;' $MYSQL_PORT1 $MYSQL_PASSWORD1
+
+	run_sql 'DROP DATABASE if exists openapi;' $MYSQL_PORT2 $MYSQL_PASSWORD2
+	run_sql 'CREATE DATABASE openapi;' $MYSQL_PORT2 $MYSQL_PASSWORD2
+}
+
+function test_source() {
+	echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>START TEST OPENAPI: SOURCE"
+	prepare_database
+	# create source succesfully
+	$cur/client/source.py "create_source1_success"
+
+	# recreate source will failed
+	$cur/client/source.py "create_source_failed"
+
+	# get source list success
+	$cur/client/source.py "list_source_success" 1
+
+	# delete source success
+	$cur/client/source.py "delete_source_success" "mysql-01"
+
+	# after delete source, source list should be empty
+	$cur/client/source.py "list_source_success" 0
+
+	# re delete source failed
+	$cur/client/source.py "delete_source_failed" "mysql-01"
+
+	# send request to not leader node
+	$cur/client/source.py "list_source_with_redirect" 0
+
+	echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>TEST OPENAPI: SOURCE SUCCESS"
+}
+
+function run() {
+	if [[ $(pip list) == *"requests"* ]]; then
+		echo "requests already installed"
+	else
+		sudo pip install requests # in ci need sudo to install requests
```

Comment:
we should install the requirement in Makefile

Reply:
At the moment only the integration tests for openapi need this dependency, I think it would be easier to read in a separate test file?😬

Reply:
you can call `make install_python_dep` in test scripts 🤔